zwave_js_server/model/driver.py

```diff
@@ -39,19 +40,40 @@ def receive_event(self, event: Event) -> None:
 
         self.emit(event.type, event.data)
 
+    def handle_logging(self, event: Event) -> None:
+        """Process a driver logging event."""
+        event.data["log_message"] = LogMessage(cast(LogMessageDataType, event.data))
+
     def handle_all_nodes_ready(self, event: Event) -> None:
         """Process a driver all nodes ready event."""
 
+    async def async_send_command(
```

Comment:
Should this be a private method?

Reply:
should we do the same for the node method?

Reply:
I haven't reviewed our use of it. If we don't use it in the integration, I think we can make it private.